test/runtests.jl

```diff
@@ -7,19 +7,19 @@ using HTTP
 
 
 @testset "HTTP" begin
-    include("utils.jl");
-    include("sniff.jl");
-    include("uri.jl");
-    include("url.jl");
-    include("cookies.jl");
-    include("parser.jl");
+    println("running utils.jl tests..."); include("utils.jl");
+    println("running sniff.jl tests..."); include("sniff.jl");
+    println("running uri.jl tests..."); include("uri.jl");
+    println("running url.jl tests..."); include("url.jl");
+    println("running cookies.jl tests..."); include("cookies.jl");
+    println("running parser.jl tests..."); include("parser.jl");
 
-    include("loopback.jl");
-    include("WebSockets.jl");
-    include("messages.jl");
+    println("running loopback.jl tests..."); include("loopback.jl");
+    # println("running WebSockets.jl tests..."); include("WebSockets.jl");
```

Comment:
?

Reply:
passing in #303 CI
https://travis-ci.org/JuliaWeb/HTTP.jl/jobs/429938842#L1158-L1168